core/src/test/java/org/opensearch/sql/expression/operator/convert/TypeCastOperatorTest.java

```diff
@@ -41,10 +43,16 @@
 import org.opensearch.sql.expression.DSL;
 import org.opensearch.sql.expression.FunctionExpression;
 import org.opensearch.sql.expression.config.ExpressionConfig;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
 
+@ExtendWith(SpringExtension.class)
+@ContextConfiguration(classes = {ExpressionConfig.class, AnalyzerTestBase.class})
 class TypeCastOperatorTest {
 
-  private final DSL dsl = new ExpressionConfig().dsl(new ExpressionConfig().functionRepository());
+  @Autowired
+  private DSL dsl;
```

Comment:
Can you name it in uppercase to avoid changing all depending tests?
```suggestion
  private DSL DSL;
```